src/components/Form.tsx

```diff
@@ -281,13 +300,25 @@ const Form: React.FC<{
                         thousandSeparator={thousandSeparator}
                         allowNegative={false}
                         valueIsNumericString
-                        onValueChange={({ value }) => onChangeToValue(value)}
-                        placeholder={swapMode === 'ExactOut' ? 'Enter desired amount' : ''}
+                        onValueChange={onChangeToValue}
+                        placeholder={swapMode === 'ExactIn' ? '' : 'Enter desired amount'}
```

Comment:
why is this changed a ? if im in exactIn mode, to token info should not have placeholder right ? 
![CleanShot 2024-06-27 at 6  59 09](https://github.com/jup-ag/terminal/assets/31162981/f87007f3-ad9b-4bbe-83da-06b491ef4f5d)